libs/npm/core/src/hooks/modal/useCustomHeight.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+/*
+ * This hook is useful for retrieving the custom height of the browser for the modal popup
+ */
+export const useCustomHeight = () => {
+  const [customHeight, setCustomHeight] = React.useState(window.innerHeight - 235);
```

Comment:
What is the hardcoded `235` from?

Reply:
Initially in the full screen mode (the height of the window's innerHeight is 955px), we set the height of the windows popup to 720px, so the difference (955-720) would be 235 in this case and based on this we could dynamically adjust the height of the window popup when resizing the browser.